theme/styles/mixins/flex.scss

```diff
@@ -0,0 +1,46 @@
+@charset "UTF-8";
+
+/// Provides a quick method for setting an element’s flex settings. Use a `null` value to “skip” a side. Fast & Neat.
+///
+/// @example sass - Usage
+///   .element1 {
+///     +flex(space-between, center);
```

Comment:
co dát jako výchozí hodnoty `center, center` a vynechávat je až při null hodnotě?

Reply:
Nemyslím si. Ten mixin by se měl chovat ve směs stějně, jako `display`:`flex`. Takže dodržovat stejné defaultní hodnoty. Celkově tedy v jakémkoli zápisu nahrazovat za `+flex()`. Možná hodnoty `center, center` jsou nejpoužívanější ( i když o tom pochybuji ), nicméně to logicky nedává smysl. Co by se stalo, kdyby člověk poprvé používal mixin a chtěl by pouze zarovnat `justify-content` na `center`?  - Najednou by se mu změnil i `align-items` na `center` ( nevyžádané chování, které potom bude muset opravovat pomocí null... )

Reply:
Kouknu na projekty, jak hodně používám hodnoty `center, center` a podle toho usoudím.